src/parsers/parser.js

```diff
@@ -186,12 +196,14 @@ class Parser {
     parseAst () {
         const token = this.lexer().peek();
 
+        // check if the token's value is a keyword
```

Comment:
So rather than put this comment, you can refactor the code to make it clearer. You can rename kwnodes to keywordNodes and kwNode to keywordNode.